samples/Calculator/Calculator/AddTwoNumbers.cs

```diff
@@ -0,0 +1,38 @@
+﻿using NetCoreTest;
+using Xunit;
+using Xunit.Gherkin.Quick;
+
+namespace Tests
+{
+    [FeatureFile("./AddTwoNumbers.feature")]
+    public sealed class AddTwoNumbers : Feature
```

Comment:
Is this the same as this? https://github.com/Enough7/Xunit.Gherkin.Quick/blob/master/source/Xunit.Gherkin.Quick.ProjectConsumer/Addition_ForMultipleUseCases/AddTwoNumbers.cs

Sorry, maybe it was confusing, but all the samples that you see are taken from this project here (https://github.com/Enough7/Xunit.Gherkin.Quick/tree/master/source/Xunit.Gherkin.Quick.ProjectConsumer) which is part of the full Xunit.Gherkin.Quick solution.

Maybe you had some other intention? Let me know.

Reply:
Yes it is the same. I think an example .csproj-File would be nice

Reply:
An example project file is here. This project contains all the samples that are on github: https://github.com/ttutisani/Xunit.Gherkin.Quick/tree/master/source/Xunit.Gherkin.Quick.ProjectConsumer